openstudiocore/resources/model/OpenStudio.idd

```diff
@@ -1015,7 +1015,6 @@ OS:SizingPeriod:DesignDay,
        \note Humidity Indicating Type
        \note required-field if Humidity Indicating Day Schedule is not used
        \type real
-       \units BasedOnField A3
```

Comment:
@kbenne what was happening here?  OS thought this thing had units in SI and was confused it was missing an IP units?